client/src/components/previews/EventSeriesPreview.js

```diff
@@ -0,0 +1,52 @@
+import API from "api"
+import { PreviewField } from "components/FieldHelper"
+import LinkTo from "components/LinkTo"
+import { EventSeries } from "models"
+import PropTypes from "prop-types"
+import React from "react"
+import Settings from "settings"
+
+const EventSeriesPreview = ({ className, uuid }) => {
+  const { data, error } = API.useApiQuery(EventSeries.getEventSeriesQuery, {
+    uuid
+  })
+
+  if (!data) {
+    if (error) {
+      return <p>Could not load the preview</p>
+    }
+    return null
+  }
+
+  const eventSeries = new EventSeries(data.eventSeries)
+
+  const eventSeriesTitle = eventSeries.name || `#${eventSeries.uuid}`
+  return (
+    <div className={`report-preview preview-content-scroll ${className || ""}`}>
+      <h4 className="ellipsized-text">Event Series {eventSeriesTitle}</h4>
+      <div className="preview-section">
```

Comment:
Where are the `status` and `description` fields?

Reply:
It is a preview, I do not think we want to have description there in my opinion.

Reply:
Other previews show the profile/description/biography as well.

Reply:
Great, I still dont think this one should.